.gitignore

```diff
@@ -9,6 +9,7 @@ node_modules
 .vagrant
 build
 npm-debug.log
+package-lock.json
```

Comment:
Wait, isn't it explicitly recommended to version control the package-lock file haha? (I do always just constantly delete it and recreate it though ehhee.)

Reply:
I hadn't realized newer versions of npm generated this, actually, I always thought it was just a byproduct of yarn (which I was under the impression neither of you used)